packages/fast-components-react-msft/src/text-action/text-action.props.ts

```diff
@@ -56,6 +52,11 @@ export interface TextActionHandledProps
      * Callback for when text action input is blurred
      */
     onBlur?: (e?: React.FocusEvent<HTMLInputElement>) => void;
+
+    /**
```

Comment:
Does this need to be handled? I think unhandledProps covers this as it is

Reply:
I believe it does because this is an overall disabled prop that controls both the input _and_ button disabled states.

Reply:
Can you make that clear in the code comment?

Reply:
Sure 👍